static/entrypoint.js

```diff
@@ -734,4 +746,26 @@ loaderElement.addEventListener(gameloader.events.prepare, (ev) => {
 });
 
 
-configureCustomKeys(loaderElement);
\ No newline at end of file
+configureCustomKeys(loaderElement);
+
+
+// Support skewing the tracker.
+(function() {
+
+  /**
+   * Adjust Santa's location between 0-1 along his total route.
+   */
+  window.santaApp.adjustSanta = (ratio, offset = 0) => {
```

Comment:
Offset units are Millisecs, correct?

Reply:
Yeah, it's safe to assume in JS that any time unit is ms until specified otherwise.